util/codec/codec.go

```diff
@@ -352,6 +353,232 @@ func encodeHashChunkRowIdx(sc *stmtctx.StatementContext, row chunk.Row, tp *type
 	return
 }
 
+// HashChunkColumns writes the encoded value of each row's column, which of index `colIdx`, to h.
+func HashChunkColumns(sc *stmtctx.StatementContext, h []hash.Hash64, chk *chunk.Chunk, tp *types.FieldType, colIdx int, buf []byte, isNull []bool) (err error) {
+	const (
+		sizeUint64  = unsafe.Sizeof(uint64(0))
+		sizeFloat64 = unsafe.Sizeof(float64(0))
+	)
+	var (
+		b   []byte
+		row chunk.Row
```

Comment:
All the usage of `row` can be replaced by `column`.

Reply:
have done